src/Invoice.php

```diff
@@ -19,7 +19,7 @@ class Invoice
     public const ZUGFERD_BASIC = Zugferd::ZUGFERD_BASIC;
     public const ZUGFERD_CONFORT = Zugferd::ZUGFERD_CONFORT;
     public const ZUGFERD_EXTENDED = Zugferd::ZUGFERD_EXTENDED;
-    
+
```

Comment:
Consider defining constants UBL_PEPPOL = Ubl::PEPPOL.
Maybe I'm wrong, but I think The API user should not need to "use" Ubl

`
use DigitalInvoice\Invoice;
$invoice = new Invoice(....., Invoice::UBL_PEPPOL);
`

Instead of
`
use DigitalInvoice\Invoice;
use DigitalInvoice\Ubl;
$invoice = new Invoice(....., Ubl::PEPPOL);
`
Let's keep the usage simple, not requiring the user to know the package internals